### DISCUSSION TITLE
Editing Camera Group Layout

### DISCUSSION THREAD

**Discussion by 18923** [blakeblackshear/frigate#15575] [General]
15 Beta 3
FYI
While editing a Camera Group Layout the only option is "Exit Editing". It's not clear if edits will be saved or not. Suggesting a SAVE/Cancel dialog.

81Xats
![frigate](https://github.com/user-attachments/assets/9cb42e3c-901a-48af-8276-8d6cb5d8b31b)

**Comment by hawkeye217** [comment]
Edits are always saved. If you change the layout, it's immediately saved to your browser's local storage. There is no intermediate state between your old layout and your new one.
